keyboards/murcielago/rev1/keymaps/default/keymap.c

```diff
@@ -0,0 +1,91 @@
+   */
+  [BASE] = LAYOUT( /* qwerty */
+    KC_EQL,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                      KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS,
+    KC_LBRC, KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,                      KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_RBRC,
+    KC_BSLS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,                      KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,
+    KC_LSPO, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_GESC, KC_DEL,  KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSPC,
+             ALT_CAP, KC_DEL,  KC_LGUI, KC_BSPC, KC_LCTL,                   ENT_AGR, KC_SPC,  RAISE,   KC_TAB,  KC_MPLY
+  ),
+
+  /* RAISED
+   *       ,-----------------------------------------.                        ,-----------------------------------------.
+   *       | RST  |  F1  |  F2  |  F3  |  F4  | F5   |                        | F6   |  F7  |  F8  |  F9  | F10  | F11  |
+   *       |------+------+------+------+------+------|                        |------+------+------+------+------+------|
+   *       |      |      |      | PC   |      |      |                        |      |PRE W |  UP  |NXT W |      | F12  |
+   *       |------+------+------+------+------+------|                        |------+------+------+------+------+------|
+   *       |      |      |      |      |      |      |                        |      | LEFT | DOWN |RIGHT |      | BL+  |
+   *       |------+------+------+------+------+------+------.          ,------+------+------+------+------+------+------|
+   *       |      |      |      | Calc |      |      |      |          |      |      |DEL PW| Play |DEL NW|      | BL-  |
+   *       `------------------------------------------------'          `------------------------------------------------'
+   *                  |      |      |      | Del  |      |                |      |      |      |      |      |
+   *                  `----------------------------------'                `----------------------------------'
+   */
+  [RAISED] = LAYOUT( /* symbols */
+    RESET,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,                     KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,
+    _______, XXXXXXX, XXXXXXX, KC_MYCM, XXXXXXX, XXXXXXX,                   XXXXXXX, PRE_WRD, KC_UP,   NXT_WRD, XXXXXXX, KC_F12,
+    _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                   XXXXXXX, KC_LEFT, KC_DOWN, KC_RGHT, XXXXXXX, BL_INC,  
+    _______, XXXXXXX, XXXXXXX, KC_CALC, XXXXXXX, XXXXXXX, _______, _______, XXXXXXX, PRE_WDL, XXXXXXX, NXT_WDL, XXXXXXX, BL_DEC,
+             _______, _______, _______, KC_DEL,  _______,                   _______, _______, _______, _______, _______
+  )
+};
+
+void matrix_scan_user(void) {
+  if (host_keyboard_leds() & (1<<USB_LED_CAPS_LOCK)) {
+    // Backlight function is used for caps lock indication
+    backlight_enable();
+  }
+  else
+  {
+    backlight_disable();
+  }
+  
+};
```

Comment:
This isn't the best way to do this, actually. 

https://docs.qmk.fm/#/custom_quantum_functions?id=led-control

```suggestion
bool led_update_user(led_t led_state) {
    if (led_state.caps_lock) {
        backlight_enable();
    } else {
        backlight_disable();
    }
    return true;
}
```

This will be called when the host leds change, and only trigger once per change. 

In fact, you could throw the following into the keyboard c file:
```c
bool led_update_kb(led_t led_state) {
    bool res = led_update_user(led_state);
    if(res) {
        if (led_state.caps_lock) {
            backlight_enable();
        } else {
            backlight_disable();
        }
    }
    return res;
}
```

This allows user to override this functionality, and for it to be enabled in the QMK Configurator.